examples/ui/text_debug.rs

```diff
@@ -35,7 +35,7 @@ fn infotext_system(mut commands: Commands, asset_server: Res<AssetServer>) {
             },
         )
         .with_style(Style {
-            align_self: AlignSelf::FlexEnd,
+            align_self: AlignSelf::FlexStart,
```

Comment:
We might be able to remove this, the default might look the same visually.

Reply:
This one looks the same without the align_self. Removed!